packages/eslint-config/package.json

```diff
@@ -14,10 +14,12 @@
     "@typescript-eslint/parser": "^2.33.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-eslint-plugin": "^2.2.1",
-    "eslint-plugin-mocha": "^7.0.0"
+    "eslint-plugin-mocha": "^7.0.0",
+    "eslint-plugin-prettier": "^3.1.3"
   },
   "peerDependencies": {
-    "eslint": "^7.0.0"
+    "eslint": "^7.0.0",
+    "prettier": "^2.0.5"
```

Comment:
I am not sure if it's a good idea to force all LB projects to check prettier formatting via eslint. It makes me wonder how easy or difficult it would be to enable the new setup only for loopback-next monorepo?

Reply:
I'll look into it. 👍